djangocms_googlemap/static/djangocms_googlemap/js/djangocms.googlemaps.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright https://github.com/divio/djangocms-googlemap
+ */
+
+var djangocms = window.djangocms || {};
```

Comment:
@vxsx we have to find a generic way of name spacing the addons (also in light of the LTS cleanups) WDYT of this approach?